distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -452,7 +453,7 @@ type Table
              people.filter "age" (age -> (age%10 == 0))
     @column Widget_Helpers.make_column_name_selector
     filter : (Column | Text | Integer) -> (Filter_Condition|(Any->Boolean)) -> Problem_Behavior -> Table ! No_Such_Column | Index_Out_Of_Bounds | Invalid_Value_Type
-    filter self column filter=(Filter_Condition.Equal True) on_problems=Report_Warning = case column of
+    filter self column (filter : Filter_Condition | Function = Filter_Condition.Equal True) on_problems=Report_Warning = case column of
```

Comment:
This has to be `Filter_Condition | Function` instead of `Filter_Condition|(Any->Boolean)` due to #7289